src/logic/utils.ts

```diff
@@ -36,3 +36,12 @@ export function getAverageWinTurn(player: Player) {
     // Return as string rounded to nearest tenth
     return average.toFixed(1);
 }
+
+export function isNewlyQualifiedPlayer(player: Player) {
+    const dateOffset = new Date();
+    dateOffset.setDate(dateOffset.getDate() - NEW_PLAYER_HIGHLIGHT_DAYS)
+    const lastIndex = player.matches.length - 1
+    if (PLAYER_MINIMUM_GAMES_REQUIRED <= player.matches.length && player.matches.length <= 15 && dateOffset < player.matches[lastIndex].date) {
```

Comment:
return false otherwise, right now there is no return if this statement is false.